styles/main.css

```diff
@@ -27,6 +31,10 @@ ul {
   margin: 0;
 	display: grid;
   place-items: center;
+  
+}
+
+.content-box {
```

Comment:
I like this, but this will clip Header element and border. 

As-is:
![image](https://user-images.githubusercontent.com/2367864/131761867-2a100da9-8911-4893-8c08-ad2bc2d26868.png)



With padding-left: 8px
![image](https://user-images.githubusercontent.com/2367864/131761850-4ab9f7a9-710c-4512-83b2-61a73b1942da.png)

padding-left could help fix this, but there could be better ways to solve this entirely I don't know about.